src/controllers/controllerpresetinfoenumerator.cpp

```diff
@@ -57,7 +57,7 @@ void PresetInfoEnumerator::loadSupportedPresets() {
     m_bulkPresets.clear();
 
     for (const QString& dirPath : m_controllerDirPaths) {
-        QDirIterator it(dirPath);
+        QDirIterator it(dirPath, QDirIterator::Subdirectories);
```

Comment:
This can is unrelated, right? I think enabling support for controller scripts in submodules is a different matter.

Reply:
It was very simple to implement and I don't think there's any reason to remove it now.